index.html

```diff
@@ -253,12 +253,13 @@ <h3 class="contact-text">I'm always interested in hearing about new projects, so
                             <label for="email" class="label">Email</label>
                         </li>
                         <li> 
-                            <textarea class="area" rows="5" maxlength=500 name="message">Enter text here</textarea>
+                            <textarea class="area" rows="5" maxlength=500 name="message" placeholder="Enter text here"></textarea>
                         </li>
                         <li>
                             <input type="submit" name="submit" value="Get in touch" class="submit">
```

Comment:
- [ ] Kindly change the `name` attribute of this input element from `submit` to anything else, for example, `btnSubmit` so we can have the expected behavior. 😎️